components/leaflet-maps/NormalMap.vue

```diff
@@ -178,7 +178,53 @@
        })
 
        return optionalsWithValue
-     }
+     },
+     getFormatDate: function (date,precision){
```

Comment:
Is there a way to import this function so we aren't copying and maintaining two versions? Hopefully there is a way for vue component files to use common functions to avoid duplication. Thanks Ruby

Reply:
Moved to a new file now. Thank you